e2e_testing/xfail_sets.py

```diff
@@ -12,7 +12,12 @@
 
 from torch_mlir_e2e_test.test_suite import COMMON_TORCH_MLIR_LOWERING_XFAILS
 
-REFBACKEND_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS
+REFBACKEND_XFAIL_SET = COMMON_TORCH_MLIR_LOWERING_XFAILS | {
+    "UnsafeView1DFoldModule_basic",
+    "View1DFoldModule_basic",
+    "ViewCollapseInferredDimModule_basic",
+    "ViewExpandInferredDimModule_basic",
```

Comment:
Thanks for creating issue https://github.com/llvm/torch-mlir/issues/1635 for these XFAILs!